src/main/flight/pid.c

```diff
@@ -418,7 +418,7 @@ static FAST_RAM_ZERO_INIT float acroTrainerGain;
 
 void pidUpdateAntiGravityThrottleFilter(float throttle)
 {
-    if (antiGravityMode) {
+    if ((antiGravityMode == ANTI_GRAVITY_SMOOTH) && antiGravityEnabled) {
```

Comment:
Should be:
```
    if (antiGravityMode == ANTI_GRAVITY_SMOOTH) {
```

Reply:
antiGravityEnabled is set by feature

Reply:
No, it can be set by feature or by mode switch if the feature is disabled.
```
    pidSetAntiGravityState(IS_RC_MODE_ACTIVE(BOXANTIGRAVITY) || feature(FEATURE_ANTI_GRAVITY));
```

Reply:
oh...pilot flips switch...and filter is empty

Reply:
Yes exactly.  If the function "can" be activated then we need to feed the filter all the time.

Reply:
Sorry, that was my bad, I did not consider the implications of halting filtering when anti gravity isn't enabled.